bench-plotter/src/bench_plotter/plotting/dashboard_processor.py

```
@@ -0,0 +1,1193 @@
+        List of panel dictionaries
+    """
+    if panels_spec is not None:
+        return panels_spec
+
+    try:
+        from bench_plotter.dashboard_queries import (
+            get_dashboard_panels as _load_panels,
+        )
+
+        return _load_panels()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)